mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -971,7 +977,7 @@ class LogisticRegression @Since("1.2.0") (
 
     val maxMemUsage = (actualBlockSizeInMB * 1024L * 1024L).ceil.toLong
     val blocks = InstanceBlock.blokifyWithMaxMemUsage(scaled, maxMemUsage)
-      .persist(StorageLevel.MEMORY_AND_DISK)
+      .persist(intermediateRDDStorageLevel)
```

Comment:
I only changed one place to demonstrate the idea. I'd like to collect the feedback before extending the change to the whole ml module.

Reply:
about 30 places based on IDEA search results.

Reply:
For me, this PR looks reasonable enough. Thank you, @pan3793 .

Reply:
@dongjoon-hyun thank you, I will complete the change tomorrow if no objections from others.